From be4a6b4af14653d5f68dea7d57d111f5eeddc59a Mon Sep 17 00:00:00 2001
From: Kartatz <105828205+Kartatz@users.noreply.github.com>
Date: Fri, 3 Jul 2026 17:44:06 +0000
Subject: [PATCH] Rename __unused to __unused0

---
 bits/stat.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/bits/stat.h b/bits/stat.h
index e4c0b24..0f2a3f3 100644
--- a/bits/stat.h
+++ b/bits/stat.h
@@ -103,7 +103,7 @@ struct stat
     __syscall_ulong_t st_ctimensec;	/* Nsecs of last status change.  */
 #endif
 #ifdef __x86_64__
-    __syscall_slong_t __unused[3];
+    __syscall_slong_t __unused0[3];
 #else
 # ifndef __USE_FILE_OFFSET64
     unsigned long int __unused4;
@@ -161,7 +161,7 @@ struct stat64
     __syscall_ulong_t st_ctimensec;	/* Nsecs of last status change.  */
 # endif
 # ifdef __x86_64__
-    __syscall_slong_t __unused[3];
+    __syscall_slong_t __unused0[3];
 # else
     __ino64_t st_ino;			/* File serial number.		*/
 # endif
-- 
2.54.0

